Tools/MediaDriverTools/KrnToHex_IGA/KrnToHex_IGA.cpp

```diff
@@ -47,8 +47,8 @@ bool KrnToHex(char *file_krn, char *file_hex)
     char     line[40];
     FILE     *f_in, *f_out;
 
-    unsigned long   *pbuffer;
-    unsigned long    n_read;
+    unsigned int   *pbuffer;
```

Comment:
Can you, please add Fixes: #452 to the commit message of this patch:
```
[VP] Correct kernel

Fixes: #452 
```

This will automatically close the issue and associate commit with it.

Reply:
Sure, I think Lin will help add "Fixes: #452" later.